src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -6,3 +6,96 @@
 from azure.cli.help_files import helps #pylint: disable=unused-import
 
 #pylint: disable=line-too-long
+
+helps['network dns record-set create'] = """
```

Comment:
>elps['network dn [](start = 1, length = 16)

Since Johan has been asking me, can you get adequate help for your create command by "stealing" the docstring from the SDK's record type object?